lib/livebook_web/live/session_live.ex

```diff
@@ -419,6 +419,13 @@ defmodule LivebookWeb.SessionLive do
             <div style="height: 80vh"></div>
           </div>
         </div>
+        <div class="absolute w-1/2 p-4 right-0" data-el-output-panel>
+          <iframe
+            class="h-full w-full"
+            src={~p"/sessions/#{@session.id}/external-window?type=output-panel&embedded=true"}
+          />
+          <.output_panel_menu />
+        </div>
```

Comment:
It is, otherwise this will break certain assumptions, like element ids, output pruning. And also we want to make the embedded and pop-out view consistent, in the pop-out view changes to form inputs are local, so we want them to be local to the embedded view as well, and with iframe we get all of these semantics for free :)

Reply:
ahh I see :+1: